chemprop/cli/predict.py

```diff
@@ -277,11 +277,11 @@ def make_prediction_for_models(
 
         # TODO: might want to write a shared function for this as train.py might also want to do this.
         preds = torch.concat(predss, 0)
-        if isinstance(model.predictor, MulticlassClassificationFFN):
-            preds = torch.argmax(preds, dim=-1)
         individual_preds.append(preds)
 
     average_preds = torch.mean(torch.stack(individual_preds).float(), dim=0)
+    if isinstance(model.predictor, MulticlassClassificationFFN):
+        average_preds = torch.argmax(average_preds, dim=2)
```

Comment:
```suggestion
        average_preds[..., 0] = torch.argmax(average_preds[..., 0], dim=-1)
```

Reply:
This wouldn't work because the dimension doesn't match.

Reply:
I see. What I mean is that the `argmax` should only operate on the first element in the last dimension, as the length in the last dimension, `s`, represents the number of targets for the predictor. Again, the current way is fine as the `s` is always `1` for the multiclass classification with the predictors we have.

Reply:
Also, as we personally discussed, do you think we should switch to provide the probability as output like binary classification?

Reply:
I think so, because that way it's consistent with the binary class prediction, and it's the most useful output form for the user to perform any analysis

Reply:
But I think this will have to be a separate PR because it's out of scope of this one.

Reply:
Yes, I will take care of it.

Reply:
I have opened an issue to track (https://github.com/chemprop/chemprop/issues/979#issue-2454762815).